app/controllers/api/v2/infrastructures_controller.rb

```diff
@@ -104,14 +104,17 @@ def profile_by_cluster_name
     render json: {
       name: @helm_infrastructure.app_group_name,
       app_group_name: @helm_infrastructure.app_group_name,
+      app_group_id: @helm_infrastructure.app_group_id,
       app_group_secret: @helm_infrastructure.app_group_secret,
       capacity: @helm_infrastructure.helm_cluster_template.name,
+      cluster_id: @helm_infrastructure.id,
```

Comment:
can we utilize `cluster_name` instead of adding new cluster_id? since it is also unique.